web/packages/shared/components/Notification/Notification.tsx

```diff
@@ -171,13 +175,50 @@ function getRenderedContent(
           color="text.secondary"
           css={longerTextCss}
         >
+          {content.list && <List items={content.list} />}
           {content.description}
+          {content.link && (
+            <Link
+              css={`
+                display: block;
+              `}
+              href={content.link.href}
+              target="_blank"
+            >
+              {content.link.text}
+            </Link>
+          )}
         </Text>
       </Flex>
     );
   }
 }
 
+function List(props: { items: string[] }) {
+  if (props.items.length === 1) {
+    return <>{props.items[0]}</>;
```

Comment:
There's no space between the single item and the notification description. Perhaps we should get rid of it? I feel like the decision whether to display something as a single text block vs a list should be left to the callsite rather than being handled deep in the Notification component.

Reply:
I'm fine with removing it, I added it as a "nice to have" feature :)